providers/websockets/okx/messages.go

```diff
@@ -24,21 +24,18 @@ type (
 const (
 	// OperationSubscribe is the operation to subscribe to a channel.
 	OperationSubscribe Operation = "subscribe"
-)
-
-const (
 	// IndexTickersChannel is the channel for mark price updates.
 	IndexTickersChannel Channel = "index-tickers"
-)
-
-const (
 	// EventSubscribe is the event denoting that we have successfully subscribed to a channel.
 	EventSubscribe EventType = "subscribe"
 	// EventTickers is the event for tickers. By default, this field will not be populated
 	// in a properly formatted message. So we set the default value to an empty string.
 	EventTickers EventType = ""
 	// EventError is the event for an error.
 	EventError EventType = "error"
+
+	// subscriptionMsgSize is the amount of tickers to subscribe to in a single Subscribe message.
+	subscriptionMsgSize = 10
```

Comment:
arbitrary

Reply:
Stuff like this belongs in the `WebsocketConfig` IMO, this way we can have all other providers use a default value (1), but okx moves to a larger # of tickers per sub